packages/web/src/components/banner/DirectMessagesBanner.tsx

```diff
@@ -0,0 +1,51 @@
+import { useCallback, useState } from 'react'
+
+import { Client, accountSelectors } from '@audius/common'
+import { push as pushRoute } from 'connected-react-router'
+import { useDispatch } from 'react-redux'
+
+import { useSelector } from 'common/hooks/useSelector'
+import { getClient } from 'utils/clientUtil'
+import { CHATS_PAGE } from 'utils/route'
+
+import { CallToActionBanner } from './CallToActionBanner'
+
+const messages = {
+  text: 'Direct Messaging Now Available!',
+  pill: 'New'
+}
+
+const { getHasAccount } = accountSelectors
+
+const DIRECT_MESSAGES_BANNER_LOCAL_STORAGE_KEY = 'dismissDirectMessagesBanner'
+
+export const DirectMessagesBanner = () => {
+  const dispatch = useDispatch()
+  const hasAccount = useSelector(getHasAccount)
+  const [isVisible, setIsVisible] = useState(
+    !window.localStorage.getItem(DIRECT_MESSAGES_BANNER_LOCAL_STORAGE_KEY) &&
+      ((getClient() === Client.DESKTOP && hasAccount) ||
```

Comment:
If you're logged out on electron, we still show?

Reply:
yeah good catch, was thinking it would take you to signup but it doesn't